tests-arquillian/src/test/java/org/jboss/weld/tests/extensions/staticObserver/StaticLifecycleEventObserverForTestException.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2017, Red Hat, Inc., and individual contributors
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.weld.tests.extensions.staticObserver;
+
+/**
+ *
+ * Exception to distinguish {@link javax.enterprise.inject.spi.DeploymentException} related to Static Lifecycle event observer
+ * method in test.
+ *
+ * @author Antoine Sabot-Durand
+ */
+public class StaticLifecycleEventObserverForTestException extends RuntimeException {
```

Comment:
Thinking about this. it is maybe an overkill to test it like this? I think it would be sufficient to have a test asserting that:

1. Static container lifecycle observer throws definition exception
2. Other static observers in extensions work as expected

@tremes WDYT?